SCAutolib/models/gui.py

```diff
@@ -385,3 +387,24 @@ def assert_no_text(self, key: str, timeout: float = 0):
             if selection.sum() != 0:
                 raise Exception(f"The key='{key}' was found "
                                 f"in the screenshot {screenshot}")
+
+    @log_decorator
+    def check_home_screen(self, *args):
+        if len(args) == 0:
```

Comment:
nit: Out of curiosity why did you use check_home_screen(self, *args) and check the len of args afterward to have default value and not just use check_home_screen(self, polarity=True) instead?

Reply:
I tried initially but, hit an issue which I am not seeing now.  I'm guessing I had a typo somewhere that I missed but, I'm not sure.

I've switched to use check_home_screen(self, polarity=True)  since that reduces the necessary code.